src/Symfony/Component/Intl/Resources/data/timezones/en.json

```diff
@@ -151,7 +151,7 @@
         "America\/Moncton": "Atlantic Time (Moncton)",
         "America\/Monterrey": "Central Time (Monterrey)",
         "America\/Montevideo": "Uruguay Time (Montevideo)",
-        "America\/Montreal": "Canada (Montreal)",
+        "America\/Montreal": "Canada Time (Montreal)",
```

Comment:
use the `regionFormat` for country-based names, this is much more consistent with explicitly translated zones, e.g. `Coordinated Universal Time` and `Central European Time`